sim/src/sensors/RotatingRaySensor.h

```diff
@@ -40,6 +40,9 @@
 #include <mars/utils/Mutex.h>
 #include <mars/interfaces/graphics/draw_structs.h>
 
+#include <velodyne_lidar/MultilevelLaserScan.h>
```

Comment:
Please switch to the base-type which is now part of master, remove the dependancy to the velodyne scanner